core/src/cell.rs

```diff
@@ -46,8 +28,7 @@ pub struct CellMeta {
 pub struct CellMetaBuilder {
     cell_output: CellOutput,
     out_point: CellOutPoint,
-    block_info: Option<BlockInfo>,
-    cellbase: bool,
+    block_info: Option<TransactionInfo>,
```

Comment:
Should we rename the field name to `transaction_info` also?

Reply:
ok